src/applications/my-education-benefits/utils/form-submit-transform.js

```diff
@@ -0,0 +1,393 @@
+// this constant maps the values on address.js in vets.json schema from VA.gov values to LTS values
+// the lts values were found on the LTS database and LTS validates them, so we need to send correct value from here
+const countries = [
```

Comment:
This list is used to map VA.gov three letter code for each country to the equivalent codes on LTS. To figure out the codes I went into the LTS country code Database and found each two letter code for each country.

During transformation, we'll map the three letter code value that comes from va.gov country pull down menu, and map it to LTS code and send on JSON.